### PR TITLE
Added tlg0559.tlg014.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Most of Hero of Alexandria Fragmenta Heroniana

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +10 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +2533 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0559/__cts__.xml`                          | New      |
| `./data/tlg0559/tlg014/__cts__.xml`                   | New      |
| `./data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml` | New      |
| `./data/tlg0559/__cts__.xml`                          | Passing  |
| `./data/tlg0559/tlg014/__cts__.xml`                   | Passing  |
| `./data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3724)*

File: data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+<body>
+ <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg0559.tlg014.1st1K-grc1">
+<pb n="179"/>
+<head>I. Fragmenta de mensuris porrectis et
+quadratis.</head> 
+<div type="textpart" subtype="chapter" n="4">
+<head>4. Tabula Heroniana I.
+<lb n="20"/> (Vide prol. § 15—20.)</head>
+<p><del>Ἥρωνοϲ εἰϲαγωγαί.</del></p>
+<p>1. Ἡ πρώτη γεωμετρία, καθὼϲ ἡμᾶϲ ὁ παλαιὸϲ
```
Comment (planatheisa) [inline data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml]: Would it not make sense to add another div-level (probably sections)?

File: data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+         <name>Frederick Hultsch</name>
+        </persName>
+       </editor>
+       <author ref="urn:cts:greekLit:tlg0559">Hero of Alexandria</author>
+       <imprint>
+        <publisher>Teubner</publisher>
+        <pubPlace>Leipzig</pubPlace>
+        <date>1864</date>
+       </imprint>
+      </monogr>
```
Comment (planatheisa) [inline data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml]: Scan missing.

Comment (AlisonBabeu) [reply data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml]: Hi @sonofmun  and @planatheisa, you are also missing the Volume number here, but you have it in the file for Julius Pollux.

**Comment by sonofmun** [comment]
As far as I can tell, this is not a multivolume edition, so I have removed it from the Julius Pollux file.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |     +144 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +2533 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0559/tlg014/__cts__.xml`                   | New      |
| `./data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml` | New      |
| `./data/tlg0559/tlg014/__cts__.xml`                   | Passing  |
| `./data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3784)*

File: data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml [hunk 3]
```diff
@@ -61,13 +61,15 @@
...
       </monogr>
+      <ref target="https://archive.org/details/metrologicorums03hultgoog">The Internet Archive</ref>
      </biblStruct>
     </listBibl>
    </sourceDesc>
   </fileDesc>
   <encodingDesc>
    <p>Text encoded in accordance with the latest EpiDoc standards</p>
    <refsDecl n="CTS">
+    <cRefPattern matchPattern="(.+).(.+)" n="chapter" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
```
Comment (planatheisa) [inline data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml]: Shouldn't this be `section`?

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |     +144 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +2533 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0559/tlg014/__cts__.xml`                   | New      |
| `./data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml` | New      |
| `./data/tlg0559/tlg014/__cts__.xml`                   | Passing  |
| `./data/tlg0559/tlg014/tlg0559.tlg014.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3810)*

**Pull request merged**